src/EnergyPlus/OutputReportPredefined.cc

```diff
@@ -1885,6 +1887,7 @@ namespace OutputReportPredefined {
         pdchPumpFlow = newPreDefColumn(pdstPump, "Water Flow [m3/s]");
         pdchPumpPower = newPreDefColumn(pdstPump, "Electric Power [W]");
         pdchPumpPwrPerFlow = newPreDefColumn(pdstPump, "Power Per Water Flow Rate [W-s/m3]");
+        pdchPumpEndUse = newPreDefColumn(pdstPump, "End Use Subcategory");
```

Comment:
Was there a reason for putting the subcategory before the motor efficiency in the table?  Looks a little odd to me to separate the motor efficiency from the other power-related columns.

Reply:
OK, I moved it to the last column.